partcad-cli/src/partcad_cli/click/commands/import.py

```
@@ -0,0 +1,48 @@
+    type=str,
+    help="Optional description for the imported part.",
+    required=False,
+)
+@click.pass_obj
+def cli(ctx, kind, source_path, desc):
+    """
+    CLI command to import a part by copying and adding to the project.
+    """
+    project = ctx.get_project(pc.ROOT)
+    if not project:
+        raise click.UsageError("Failed to retrieve the project.")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)